src/components/Organisms/Header/Nav/Nav.js

```diff
@@ -110,19 +102,17 @@ const MainNav = ({ navItems }) => {
                     </NavLink>
                   </Text>
                 ) : (
-                  <Text>
-                    <NavLink
-                      href={thisUrl}
-                      inline
-                      aria-expanded={!!isSubMenuOpen[group.id]}
-                      aria-haspopup={hasPopUp}
-                      onClick={toggleSubMenu(group.id, group.links)}
-                      onKeyUp={keyPressed(group.title)}
-                      role="button"
-                    >
-                      {thisFirstChild.title}
-                    </NavLink>
-                  </Text>
+                  <NavLink
+                    href={hasPopUp ? '#' : thisUrl}
+                    inline
+                    aria-expanded={!!isSubMenuOpen[group.id]}
+                    aria-haspopup={hasPopUp}
+                    onClick={hasPopUp ? e => toggleSubMenu(e, group.id) : null}
+                    onKeyUp={keyPressed(group.title)}
+                    role="button"
+                  >
+                    {thisFirstChild.title}
+                  </NavLink>
```

Comment:
I know it's not due to your changes but I find this whole block very hard to parse:

```
{!isMobile ? (
                  <Text>
                    <NavLink
                      href={thisUrl}
                      inline
                      rel={relNoopener}
                      aria-haspopup={hasPopUp}
                      onKeyUp={keyPressed(group.title)}
                    >
                      {thisFirstChild.title}
                    </NavLink>
                  </Text>
                ) : (
                  <NavLink
                    href={hasPopUp ? '#' : thisUrl}
                    inline
                    aria-expanded={!!isSubMenuOpen[group.id]}
                    aria-haspopup={hasPopUp}
                    onClick={hasPopUp ? e => toggleSubMenu(e, group.id) : null}
                    onKeyUp={keyPressed(group.title)}
                    role="button"
                  >
                    {thisFirstChild.title}
                  </NavLink>
                )}
```

Do we need the ternary or could just have one `NavLink` element with conditional props?

Reply:
I've taken some unnecessary logic already .. I got a bit apprehensive touching it more .. that's definitely something we need to get back to. I'm working with Ruby on the new Header, hopefully we can replace it soon.

Reply:
Ok well at the very least ... would be good to flip the ternary if poss? 

I always find `!something ? ... : ...` hard to follow

Reply:
Thanks!